### ISSUE TITLE
Add Lighthouse to the CI pipeline

### ISSUE THREAD

**Issue by bfabio** [italia/designers.italia.it#530]
It would be nice to have Lighthouse automatically ran on every change request, for example using https://github.com/treosh/lighthouse-ci-action.

**Comment by Fupete** [comment]
Yes, we need to integrate also [pa11y](https://pa11y.org/) in the building/merging process, as a blocker where needed.

And, on Vercel PR previews, I think that could be nice to have [sa11y](https://sa11y.netlify.app/) active by default, for our editors to address early a11y issues. It is a cool project we can think of to "adapt" to our needs also for more general a11y and readability purposes in 2023.

**Comment by Fupete** [comment]
cc @astagi

**Comment by bfabio** [comment]
@Fupete I think this one can be closed after #636 and the improvements (pa11y + sa11y ) added as a new enhancement post-release.

**Comment by bfabio** [comment]
Reopening because we need to

- Make it work on PRs
- Improve the message, make it clear what pages are being tested and always have the home page as the main result

**Comment by bfabio** [comment]
Check that it works
